src/xdp_iphash_to_cpu_kern.c

```diff
@@ -102,55 +102,95 @@ bool parse_eth(struct ethhdr *eth, void *data_end,
 	return true;
 }
 
+static __always_inline struct ip_hash_info *get_ip_info(struct ip_hash_key *ip)
+{
+	struct ip_hash_info *ip_info;
+
+	ip_info = bpf_map_lookup_elem(&map_ip_hash, ip);
+	if (!ip_info) {
+		struct ip_hash_key null_addr;
+	        null_addr.prefixlen = 128;
+	        null_addr.address.in6_u.u6_addr32[0] = 0;
+	        null_addr.address.in6_u.u6_addr32[1] = 0;
+        	null_addr.address.in6_u.u6_addr32[2] = 0;
+	        null_addr.address.in6_u.u6_addr32[3] = 0;
+		/* On LAN side (XDP-ingress) some uncategorized traffic are
+		 * expected, e.g. services like DHCP are running and IPs
+		 * contacting captive portal (which are not yet configured)
+		 */
+		// bpf_debug("cant find ip_info->cpu id for ip:%u\n", ip);
+		// the all-zeroes address is for default traffic
```

Comment:
Here the code change the missed lookup "unknown traffic" default entry from `255.255.255.255` to `0.0.0.0`

Reply:
Resolved by https://github.com/xdp-project/xdp-cpumap-tc/pull/14/commits/085be76ec96f1fd9f114817d47435e9ec2c6f790